docs/telegram-usage.md

```diff
@@ -87,6 +87,41 @@ Example configuration showing the different settings:
    },
 ```
 
+## Create a custom keyboard (command shortcut buttons)
+
+Telegram allows us to create a custom keyboard with buttons for commands.
+The default custom keyboard looks like this.
+
+```python
+[
+    ["/daily", "/profit", "/balance"], # row 1, 3 commands
+    ["/status", "/status table", "/performance"], # row 2, 3 commands
+    ["/count", "/start", "/stop", "/help"] # row 3, 4 commands
+]
```

Comment:
I hope you're ok that i changed quotes here - using double-quotes makes it possible to copy this list to the configuration (single quotes are not valid json).

Reply:
YES. good point